.circleci/config.yml

```diff
@@ -4,56 +4,38 @@ version: 2
 jobs:
   python:
     docker:
-      - image: udata/circleci:2-alpine
+      - image: cimg/python:3.7.13
       - image: mongo:3.6
       - image: redis:alpine
     environment:
        BASH_ENV: /root/.bashrc
     steps:
       - checkout
-      - run:
-          name: Compute Python dependencies key
-          command: cat requirements/*.pip > python.deps
       - run:
           name: Get the base reference branch
           command: export BASE_BRANCH=$(base_branch)
       - restore_cache:
           keys:
-          - py3-cache-v9-{{ arch }}-{{ checksum "python.deps" }}
-          - py3-cache-v9-{{ arch }}-{{ .Branch }}
-          - py3-cache-v9-{{ arch }}-{{ .Environment.BASE_BRANCH }}
+          - py3-cache-v2-{{ arch }}-{{ checksum "poetry.lock" }}
```

Comment:
Maybe don't drop the other keys since we could have cache conflicts between branches as far as I understand

Reply:
Ok now I have to lookup how caching works 😫